### ISSUE TITLE
Unable to create a sockjs  conenction

### ISSUE THREAD

**Issue by maxdogger** [MovingImage24/JmeterSockJsSampler#1]
Hi

I am unable to establish a sockjs connection. Please refer the following screenshot.

![image](https://user-images.githubusercontent.com/10431130/46795324-7f8be400-cd67-11e8-8c75-fd1fa24f7fa5.png)

**Comment by hakim-dc** [comment]
hello maxdogge, did you find a solution ?

**Comment by maxdogger** [comment]
No. Is there something you can suggest in case I am not doing it correctly?

**Comment by hakim-dc** [comment]
No. I didn't make it too :(. Did you find any other alternative to to do this tests please ?
